src/main/java/Commands/FindCommand.java

```diff
@@ -0,0 +1,33 @@
+package Commands;
+
+import Exceptions.DukeException;
+import Tasks.TaskList;
+import Tasks.Task;
+import util.Storage;
+import util.Ui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class FindCommand extends DukeCommand {
+    public FindCommand(String description) {
+        super(description);
+    }
+
+    public String execute(TaskList tasks, Ui ui, Storage storage) throws DukeException, IOException {
+        TaskList arrList = new TaskList();
```

Comment:
I think better naming convention can be implemented here, instead of `arrList` which looks like `ArrayList`, you could name it as something like `tasksFound`.